packages/eslint-plugin-orbit-components/src/rules/useRtl.ts

```diff
@@ -2,13 +2,13 @@ import * as t from "@babel/types";
 import { Rule } from "eslint";
 
 export const ERROR_RTL_SPACING =
-  "Values for right and left edges are different. Use Orbit's rtlSpacing utility.";
+  "Use Orbit's rtlSpacing utility because values for right and left edges are different. https://orbit.kiwi/utilities/right-to-left-languages/#rtlspacing";
```

Comment:
This change causes the style of error messages to be inconsistent again, not sure if that's on purpose.

Reply:
why incosistent? i changed it, because after the last change it was showing `rtlSpacing` error where supposed to be `right/left` :)